code/modules/mob/living/status_procs.dm

```diff
@@ -292,10 +294,11 @@
 
 // PARALYSE
 
-/mob/living/Paralyse(amount, updating = 1, force = 0)
-	return SetParalysis(max(paralysis, amount), updating, force)
+/mob/living/Paralyse(amount, updating = 1, force = 0, ane = FALSE)
+	return SetParalysis(max(paralysis, amount), updating, force, ane)
 
-/mob/living/SetParalysis(amount, updating = 1, force = 0)
+/mob/living/SetParalysis(amount, updating = 1, force = 0, ane = FALSE)
```

Comment:
updating = TRUE

and in any other proc I added "ane = FALSE".
L-look, I just like to know which arguments are Booleans when I'm using a function ;--;